message CreateRoleReq
{
	required string username = 1;
	required string profession  = 2;
}

message CreateRoleRes
{
	required bool result = 1;
	optional string characterId = 2;
}